colossalai/inference/README.md

```diff
@@ -35,12 +44,14 @@ In this section we discuss how the colossal inference works and integrates with
   - [x] context forward
   - [x] token forward
   - [x] support flash-decoding
-- [ ] Replace the kernels with `faster-transformer` in token-forward stage
 - [ ] Support all models
   - [x] Llama
   - [x] Llama-2
   - [x] Bloom
   - [x] Chatglm2
+- [ ] Quantization
+  - [x] GPTQ
+  - [x] SmoothQuant
 - [ ] Benchmarking for all models
```

Comment:
Maybe we should add some details about paged attn support, continuous batching support and pipeline parallelism support